src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -200,8 +197,9 @@ describe("FunctionsEmulator-Runtime", () => {
           return {
             function_id: require("firebase-functions")
               .firestore.document("test/test")
-              // tslint:disable-next-line:no-empty
-              .onCreate(async () => {}),
+              .onCreate(() => {
```

Comment:
Why did you only change this one and not other similar blocks in the file?

Reply:
Probably 'cause they didn't cause trouble. I've cleaned up a few more.

Generally speaking, anything that checks the logs _has_ to return a promise or value. Anything that doesn't just needs to succeed (and doing absolutely nothing is fine).